docs/src/languages/php/extensions.md

```diff
@@ -60,110 +61,111 @@ You can disable those by adding them to the `disabled_extensions` list.
-| sourceguardian|     |     |     | *   | *   |     |     |     |
-| spplus        | *   | *   |     |     |     |     |     |     |
-| sqlite3       | *   | *   | *   | *   | *   | *   | *   | *   |
-| sqlsrv        |     |     |     | *   | *   | *   |     | *   |
-| ssh2          | *   | *   | *   | *   | *   | *   | *   | *   |
-| sysvmsg       |     |     |     | *   | *   | *   | *   | *   |
-| sysvsem       |     |     |     | *   | *   | *   | *   | *   |
-| sysvshm       |     |     |     | *   | *   | *   | *   | *   |
-| tideways      |     |     |     | *   | *   | *   | *   | *   |
-| tideways-xhprof  |     |     |     | *   | *   | *   | *   |  *  |
-| tidy          | *   | *   | *   | *   | *   | *   | *   | *   |
-| tokenizer     |     |     |     | *   | *   | *   | *   | *   |
-| uuid          |     |     |     |     | *   | *   | *   | *   |
-| wddx          |     |     |     | *   | *   | *   | *   | *   |
-| xcache        | *   | *   |     |     |     |     |     |     |
-| xhprof        | *   | *   | *   |     |     |     |     |     |
-| xml           |     |     |     | *   | *   | *   | *   | *   |
-| xmlreader     |     |     |     | *   | *   | *   | *   | *   |
-| xmlrpc        | *   | *   | *   | *   | *   | *   | *   | *   |
-| xmlwriter     |     |     |     | *   | *   | *   | *   | *   |
-| xsl           | *   | *   | *   | *   | *   | *   | *   | *   |
-| yaml          |     |     |     |     | *   | *   | *   | *   |
-| zbarcode      |     |     |     | *   | *   | *   | *   |     |
-| zendopcache   | *   |     |     |     |     |     |     |     |
-| zip           |     |     |     | *   | *   | *   | *   | *   |
+| Extension        | 5.4 | 5.5 | 5.6 | 7.0 | 7.1 | 7.2 | 7.3 | 7.4 | 8.0 |
+|------------------|-----|-----|-----|-----|-----|-----|-----|-----|-----|
+| amqp             |     |     |     | *   | *   | *   | *   | *   |     |
+| apc              | *   | *   |     |     |     |     |     |     |     |
+| apcu             | *   |     | *   | *   | *   | *   | *   | *   | *   |
+| apcu_bc          |     |     |     | *   | *   | *   | *   | *   |     |
+| applepay         |     |     |     | *   | *   |     |     | *   |     |
+| bcmath           |     |     |     | *   | *   | *   | *   | *   | *   |
+| blackfire        | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| bz2              |     |     |     | *   | *   | *   | *   | *   | *   |
+| calendar         |     |     |     | *   | *   | *   | *   | *   | *   |
+| ctype            |     |     |     | *   | *   | *   | *   | *   | *   |
+| curl             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| dba              |     |     |     | *   | *   | *   | *   | *   | *   |
+| dom              |     |     |     | *   | *   | *   | *   | *   | *   |
+| enchant          | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| event            |     |     |     |     | *   | *   | *   | *   |     |
```

Comment:
I'm not actually sure what this one is...

Reply:
This one is third party its merged but not released or activated on the image will work on that

Reply:
I've updated the image and the doc will be included on the next php8 image release